src/utils/theme.js

```diff
@@ -30,9 +31,21 @@ export const breakpoints = {
   desktop: 1197
 }
 
+export const elementSizes = {
+  tappableArea: 48
+}
+
+export const zIndexes = {
+  header: 999,
+  modal: 9999,
```

Comment:
maybe change modal to 1000 and grid debugger to 1001

Reply:
see, now I can change them all in the same file 😄